code/modules/research/designs/mechfab_designs.dm

```diff
@@ -913,51 +913,52 @@
 	name = "industrial suit control module assembly"
 	desc = "An assembly for a heavy, powerful rig used by construction crews and mining corporations."
 	id = "rig_industrial"
-	materials = list(DEFAULT_WALL_MATERIAL = 30000, "glass" = 12500)
```

Comment:
this would require industrial RIG and eva RIG to use silver. As it would use parent's definition for `/datum/design/rig` , is that intentional?

Reply:
I have to concur here.
With the recent capability removal from the industrial / eva rig it is fine if they can be made using only basic materials.

Reply:
Fixed it then.